components/docs-chef-io/content/automate/configuration.md

```diff
@@ -31,7 +31,7 @@ This section describes those settings and how to change them on an existing Chef
 
 #### Chef Automate FQDN
 
-To change the fully qualified domain name (FQDN) of your Chef Automate installation, create a TOML file that contains the partial configuration:
+The maximum size of a customized fully qualified domain name (FQDN) is 25 characters. To change the  of your Chef Automate installation, create a TOML file that contains the partial configuration:
```

Comment:
Good guess, but, well, not so much. There is no limit on the size of the whole FQDN string.
The 25-character limit is only _after_ the final dot, so something like this is invalid:
`http://www.foo.this-portion-is-probably-too-long`

Reply:
@SEAjamieD So 25 characters for the top-level domain?

http://www.foo.this-portion-is-probably-too-long
<scheme>://<subdomain>.<second-level domain>.<top-level domain>

Reply:
@kagarmoe - Correct, thank you!